wallet/src/main/java/com/android/identity_credential/wallet/logging/MdocPresentationEvent.kt

```diff
@@ -0,0 +1,14 @@
+package com.android.identity_credential.wallet.logging
+
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+
+data class MdocPresentationEvent (
+    override var timestamp: Instant = Clock.System.now(),
+    override var eventId: Long = EventLogger.generateUniqueID(),
+    override var documentId: String = "",
+    override var eventType: EventLogging,
+    var presentationType: PresentationType? = null,
```

Comment:
I'm not sure what you're trying to do with `PresentationType` or why it can be null or why it doesn't have proximity (right now it has the values `OpenID4VP` and `CRED_MAN` but nothing about proximity).

I also think I previously suggested we should include `sessionTranscript` so I would just include that and not bother with `PresentationType`.... this is because with `sessionTranscript` we can easily figure out if this is a proximity presentation which is something we need when presenting the event in the UI.

So just delete `presentationType` and include `sessionTranscript` and I think it's all good?

Reply:
I thought we could infer proximity from the presentationType (OpenID4VP, CRED_MAN) but it looks like that is not correct. Replaced with sessionTranscript.